health-services/plan-service/src/main/java/digit/web/models/facility/FacilitySearchRequest.java

```diff
@@ -0,0 +1,20 @@
+package digit.web.models.facility;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.egov.common.contract.request.RequestInfo;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class FacilitySearchRequest {
+    @JsonProperty("RequestInfo")
+    private RequestInfo requestInfo;
+
+    @JsonProperty("Facility")
+    private FacilitySearchCriteria facilitySearchCriteria;
```

Comment:
I understand the point about naming clarity. However, in this case, we are passing the property as 'Facility' in the search criteria JSON, and changing it to 'FacilitySearchCriteria' might confuse the existing API consumers who expect 'Facility' as the key. Since this is part of the API contract and already in use, we prefer to keep the name as is for consistency.